src/wrappers/cache.js

```diff
@@ -1,14 +1,50 @@
 import ms from 'ms';
 import debug from 'debug';
 import memoizer from 'lru-memoizer';
+import { existsSync, readFile, writeFile, writeFileSync } from 'fs';
 
-export default function(client, { cacheMaxEntries = 5, cacheMaxAge = ms('10m') } = options) {
+export default function (client, { cacheMaxEntries = 5, cacheMaxAge = ms('10m'), useTmpFileCache = false } = options) {
   const logger = debug('jwks');
   const getSigningKey = client.getSigningKey;
 
   logger(`Configured caching of singing keys. Max: ${cacheMaxEntries} / Age: ${cacheMaxAge}`);
+
+  const fileCacheGet = (kid, callback) => {
+    const filePath = '/tmp/jwks-cache';
+    if (!existsSync(filePath)) {
+      writeFileSync(filePath, JSON.stringify({}));
+    }
+    readFile(filePath, 'utf8', (err, data) => {
+      if (err) {
+        return callback(err);
+      }
+      const jsonData = JSON.parse(data);
+      if (jsonData[kid]) {
+        return callback(null, jsonData[kid]);
+      }
+
+      getSigningKey(kid, (err, key) => {
+        if (err) {
+          return callback(err);
+        }
+        logger(`Caching signing key in filesystem for '${kid}':`, key);
+        const content = { ...jsonData, [kid]: key };
+        writeFile(filePath, JSON.stringify(content), (err) => {
```

Comment:
Should this use `writeFileSync` instead? because if the lambda runtime is stopped before the `writeFile` process, the cache may not able to write into the `/tmp` folder